docs/eos-learn/eosio-components.md

```diff
@@ -53,6 +63,7 @@ This installation guide will aim for Windows users, unfortunately, EOSIO does no
 - Now you need to restart your Windows OS
 - Finally , go to your *Microsoft Store* and Search for whatever Linux OS you want, we recommend you install Ubuntu 18.04 LTS
```

Comment:
> Finally ,

Extra space

Reply:
we can use the 👾 Space Invader 👾 emoji when we find an extra space that needs to be removed